Dockerfile

```diff
@@ -89,10 +89,25 @@ RUN set -x && apt-get update \
   && cd /opt/intel/crypto-api-toolkit \
   # disable building tests
   && sed -i -e 's;test;;g' ./src/Makefile.am \
+  # disable enclave signing inside CTK
+  && sed -i -e '/libp11SgxEnclave.signed.so/d' ./src/p11/trusted/Makefile.am \
   && ./autogen.sh \
   && ./configure --enable-dcap --with-token-path=/home/tcs-issuer \
   && make && make install
 
+# Sign the enclave with custom config.
+COPY enclave-config enclave-config
+ENV SGX_SIGN=/opt/intel/sgxsdk/bin/x64/sgx_sign
```

Comment:
Is the idea that someone might want to bring their own sgx_sign binary?

Reply:
No, the sgx_sign is available from SGX SDK, I do not expect any custom tool for enclave signing, Instead, someone could use their own tool for signing the material generated by the 'sgx_sign' in the first step.

Reply:
But why do you need SGX_SIGN env variable then?

Reply:
Just to reuse the binary path. This is a build container, so the environment variable does not end up in the final image.